Index: new.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print('Wellcome SoftUni')\r\nprint('New')\r\nprint('How are you')\r\nprint('Better than you')\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/new.py b/new.py
--- a/new.py	(revision e2b9c8de7a065ffadb5770a1e9a79a0c606f3daf)
+++ b/new.py	(date 1690807381695)
@@ -2,3 +2,4 @@
 print('New')
 print('How are you')
 print('Better than you')
+print('What?')
